crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -0,0 +1,369 @@
+use std::collections::HashMap;
+
+use masking::Secret;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    connector::utils::{PaymentsAuthorizeRequestData, RouterData},
+    core::errors,
+    services,
+    types::{self, api, storage::enums},
+};
+
+#[derive(Debug, Clone, Serialize, PartialEq)]
+pub struct IatapayAuthUpdateRequest {
+    grant_type: String,
+    scope: String,
+}
+impl TryFrom<&types::RefreshTokenRouterData> for IatapayAuthUpdateRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(_item: &types::RefreshTokenRouterData) -> Result<Self, Self::Error> {
+        Ok(Self {
+            grant_type: "client_credentials".to_string(),
+            scope: "payment".to_string(),
```

Comment:
can you add comments here to give the context on this refresh token request

Reply:
added